### PR TITLE
Add KDE Connect

### PR DESCRIPTION
**Pull request by nightscape**

### CODE REVIEW

**Comment by Ash258** [comment]
I am not fan of these all nightly builds apps inside extras bucket. Do they provide some stable build? Or just nightly version?

File: bucket/kdeconnect.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+        ]
+    ],
+    "checkver": {
+        "url": "https://binary-factory.kde.org/job/kdeconnect-kde_Nightly_win32/",
+        "regex": "kdeconnect-kde-master-([\\d+\\.]+)-windows-.*\\.7z"
+    },
+    "autoupdate": {
+        "url": "https://binary-factory.kde.org/job/kdeconnect-kde_Nightly_win32/lastSuccessfulBuild/artifact/kdeconnect-kde-master-$version-windows-msvc2017_32-cl.7z",
+        "hash": {
+            "url": "https://binary-factory.kde.org/job/kdeconnect-kde_Nightly_win32/lastSuccessfulBuild/artifact/kdeconnect-kde-master-$version-windows-msvc2017_32-cl.7z.sha256"
```
Comment (Ash258) [inline bucket/kdeconnect.json]:
```suggestion
            "url": "$url.sha256"
```

**Comment by nightscape** [comment]
Unfortunately I haven't found any builds other than those nightlies...

**Comment by rasa** [comment]
We don't have a cron on https://github.com/scoopinstaller/nightlies, but we do on https://github.com/scoopinstaller/versions. You could submit a PR there.

**Comment by rasa** [comment]
> I am not fan of these all nightly builds apps inside extras bucket.

@Ash258 Perhaps we add https://github.com/scoopinstaller/nightlies to our cron job and move nightlies there?

**Pull request closed without merging**